sdk/trace/src/main/java/io/opentelemetry/sdk/trace/SdkSpan.java

```diff
@@ -465,11 +472,13 @@ public ReadWriteSpan recordException(Throwable exception, Attributes additionalA
       additionalAttributes = Attributes.empty();
     }
 
+    int maxAttributeLength = spanLimits.getMaxAttributeValueLength();
     AttributesMap attributes =
         AttributesMap.create(
             spanLimits.getMaxNumberOfAttributes(), spanLimits.getMaxAttributeValueLength());
 
-    AttributeUtil.addExceptionAttributes(exception, attributes::put);
+    AttributeUtil.addExceptionAttributes(
```

Comment:
So the exception attributes are still resolved eagerly and the raw exception is also appended to the event?   I guess that's fine.  If someone wanted to they could have the resolver not render the `exception.stacktrace` attribute and have a processor add one on later.